invenio_rest/exceptions.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
Probably better if we rename this file to ``errors.py`` as most other modules use this convention.